Makefile

```diff
@@ -56,6 +57,13 @@ build-env: # Make a new conda environment
 install: # Install setuptools, install this package in editable mode
 	pip install --upgrade pip setuptools
 	pip install -e .[DEV]
+	@echo "Checking if the vivarium repository has the branch $(GIT_BRANCH)..."
+	@$(eval BRANCH_EXISTS=$(shell curl -s https://api.github.com/repos/ihmeuw/vivarium/branches | grep -q '"name": "$(GIT_BRANCH)"' && echo "yes" || echo "no"))
+	@if [ "$(BRANCH_EXISTS)" = "yes" ]; then \
+		pip install git+https://github.com/ihmeuw/vivarium@${GIT_BRANCH}; \
+	else \
+		pip install git+https://github.com/ihmeuw/vivarium@main; \
```

Comment:
You don't actually need this else block, as `vivarium` is already installed when we called `pip install -e .[DEV]` above